lua/lv-barbar/init.lua

```diff
@@ -1,3 +1,28 @@
-vim.api.nvim_set_keymap('n', '<TAB>', ':BufferNext<CR>', { noremap = true, silent = true })
-vim.api.nvim_set_keymap('n', '<S-TAB>', ':BufferPrevious<CR>', { noremap = true, silent = true })
-vim.api.nvim_set_keymap('n', '<S-x>', ':BufferClose<CR>', { noremap = true, silent = true })
+local wk = require('which-key')
+local opts = {
+    mode = "n", -- normal mode
+    buffer = nil, -- global mappings. specify a buffer number for buffer local mappings
+    silent = true, -- use `silent` when creating keymaps
+    noremap = true, -- use `noremap` when creating keymaps
+    nowait = false -- use `nowait` when creating keymaps
+}
+
+local mappings = {
```

Comment:
IMO, either we centralize mappings in which-key or we put them in the specific plugin, but a mix of both is bad.

Centralizing in which key offers a 'all in one place' solution, which is nice because you only have one file to maintain.
Centralizing in the plugin file makes it easier to load/unload the plugin because the which-key file doesn't need to be updated.

I personally prefer the first option.

Reply:
I think there are merits to both approaches, when you're defining custom functions then you might want to have the mappings closer since you're more likely to make more modifications. For default mappings, it doesn't matter much I guess.

I didn't notice that the project is now following a more centralized layout, so I'll adjust the commit to keep the pattern :)